packages/react/src/VideoPlayer/VideoPlayer.tsx

```diff
@@ -57,91 +55,66 @@ const Root = ({
   playIcon: PlayIcon = () => <DefaultPlayIcon className={styles.VideoPlayer__playButton} />,
   ...rest
 }: VideoPlayerProps) => {
-  const videoWrapperRef = useRef<HTMLDivElement>(null)
-  const isSmall = useVideoResizeObserver({videoWrapperRef, className: styles['VideoPlayer__container--small']})
-
   const useVideoContext = useVideo()
-  const {ccEnabled, isPlaying, ref, togglePlaying} = useVideoContext
-
-  const [isInteracting, setIsInteracting] = useState(false)
-
-  useEffect(() => {
-    const videoWrapper = videoWrapperRef.current
-    let hideControlsTimeout: NodeJS.Timeout
-    const inactivityTimeout = 3000
-
-    if (!videoWrapper) {
-      return
-    }
-
-    const showControls = () => {
-      setIsInteracting(true)
-
-      clearTimeout(hideControlsTimeout)
-
-      hideControlsTimeout = setTimeout(() => {
-        setIsInteracting(false)
-      }, inactivityTimeout)
-    }
-
-    const hideControls = () => {
-      setIsInteracting(false)
-    }
+  const {ccEnabled, fullscreenRef, isPlaying, ref, togglePlaying} = useVideoContext
```

Comment:
Just noticed this. Can we rename `useVideoContext` to something else? It reads like a hook (function) rather than a value. Could we rename this to something else like videoContext?

Reply:
Good suggestion, thanks 👍